src/main/webapp/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="hideElements()">
+    <header>
+      <a href="index.html"><h1>Black Student Insight</h1></a>
+      <input type="text" id="school-search" placeholder="Find your institution"/>
+      <button onclick="loadSchoolInfo()">Search</button>
+    </header>
+    <div id="information">
+      <h2>Our Focus</h2>
+      <p>Black Student Insight serves as a platform for current Black college students
```

Comment:
Nit: since you break the paragraph string, start it on the line after `p` rather than the line of. I start on the same line only if it's short enough not to break to a second line.